magento/pwa-studio
# [PWA-815] Enable shoppers to change their currency using a currency switcher

## Description

Adding currency switcher component to allow visitor to switch display currency.

## Related Issue





https://jira.corp.magento.com/browse/PWA-815
https://jira.corp.magento.com/browse/PWA-858

## Acceptance 



### Verification Stakeholders

@jimbo
@sirugh
@dpatil-magento

### Specification



### Verification Steps


1. Configure Magento to have more then one allowed currencies
2. See currency switcher appeared in the PWA header for desktop / Side navigation for mobile screen
3. Use switcher to change currency
4. Verify that prices and currency changed to selected currency

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
